### PR TITLE
Support testing bigger vms

### PR DESCRIPTION
**Pull request by nirs**
Various tweaks to support testing bigger vms with more cpus, memory, and large disks.

- Add script for watching replication changes
- Allow pulling from insecure local registry
- Use registry:v2 instead of latest
- Use registry image from quay.io
- Enlarge kubevirt environment

### CODE REVIEW

File: test/registry/systemd/registry.container [hunk 1]
```diff
@@ -4,7 +4,7 @@ Description=Registry container
 [Container]
 Label=app=registry
 ContainerName=registry
-Image=docker.io/library/registry:latest
+Image=docker.io/library/registry:2
```
Comment (akalenyu) [inline test/registry/systemd/registry.container:7]: So dockerhub is usually a pain in CI, unless you have a subscription. We went through all of it a few years ago and moved stuff to quay. https://github.com/kubevirt/containerized-data-importer/pull/2664

File: test/envs/kubevirt.yaml [hunk 2]
```diff
@@ -13,10 +13,10 @@ profiles:
         io.containerd.grpc.v1.cri:
           device_ownership_from_security_context: true
     network: $network
-    cpus: 4
-    memory: "8g"
+    cpus: 8
+    memory: "16g"
     extra_disks: 1
-    disk_size: "50g"
+    disk_size: "100g"
```
Comment (akalenyu) [inline test/envs/kubevirt.yaml:19]: oh wow really? will the entire 100g get used?

File: test/envs/kubevirt.yaml [hunk 3]
```diff
@@ -13,10 +13,10 @@ profiles:
         io.containerd.grpc.v1.cri:
           device_ownership_from_security_context: true
     network: $network
-    cpus: 4
-    memory: "8g"
+    cpus: 8
+    memory: "16g"
```
Comment (akalenyu) [inline test/envs/kubevirt.yaml:17]: same as the disk suggestion, maybe consider something less aggressive than doubling? 12?

**Review by akalenyu — COMMENTED** [review]
looks good. resource req are ofc up to you but dockerhub is usually a big warning sign for CI, unless you have some subscription

File: test/registry/systemd/registry.container (hunk 1 above)
Comment (nirs) [reply test/registry/systemd/registry.container:7]: Right, this issue already exits in the current code. I can add a commit using libpod image.

File: test/envs/kubevirt.yaml (hunk 2 above)
Comment (nirs) [reply test/envs/kubevirt.yaml:19]: No, this is just virtual size. The bigger image we work with is 7g compressed qcow2/40g raw (with 13g of data). We store 13g in ceph, so maybe the 50g image is enough. I increased it so we avoid space issue.

File: test/envs/kubevirt.yaml (hunk 3 above)
Comment (nirs) [reply test/envs/kubevirt.yaml:17]: The windows server needs 8gs, and we have one cluster, so 16g is fine. This is the total memory for 3 clsuters with regional-dr.yaml.

**Comment by akalenyu** [comment]
@nirs I think you had an intention to change the registry image?

**Comment by nirs** [comment]
> @nirs I think you had an intention to change the registry image?

I can but this is not a new issue introduced by this change, so I wan to do this later. We don't use this in the CI yet, this is a sample configuration that developers can use.

**Review by akalenyu — APPROVED** [review]

**Comment by nirs** [comment]
@akalenyu I added a commit using the image you suggested, thanks!

File: test/scripts/watch-replication [hunk 4]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```
Comment (ShyamsundarR) [inline test/scripts/watch-replication:1]: I am not sure this belongs with this PR. We start with a utility and then integrate it with the flow, at which point it becomes difficult to extricate it with the overall scheme of testing and samples. Merging it for now, let's see what the future brings.

**Review by ShyamsundarR — APPROVED** [review]

File: test/scripts/watch-replication (hunk 4 above)
Comment (nirs) [reply test/scripts/watch-replication:1]: This is just a tool I added to demonstrate how to wait until changes are replicated during the demo. With openshift console we can use the failover dialog showing the last sync time.

I think something providing this functionality can be useful in the kubectl-ramen plugin in the future.

**Pull request merged**